core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -931,6 +931,11 @@
    */
   public static final String LOCAL_DICTIONARY_ENABLE_DEFAULT = "false";
 
+  /**
+   * System property to enable or disable local dictionary generation
+   */
+  public static final String LOCAL_DICTIONARY_SYSTEM_ENABLE = "carbon.local.dictionary.enable";
```

Comment:
If we already have table level property, why this system level property is required? I think too many system property will make CarbonData complex to use and configure

Reply:
this is done, beacuse if we have many tables to be created and all need to be local dictionary enable, then no need to give table level property for every table, can configure the system level property